docs/guides/abn-tests.mdx

```diff
@@ -53,59 +48,56 @@ Finally, don't forget to click on the **Save** button at the top right hand side
 
 ```jsx
 statsig.logEvent('add_to_cart', 'groceries', {'price': '9.99', 'SKU_ID': 'SKU-123'});
 ```
 
-## Step 6: Review experiment results
+As the experiment progresses, you will see how many experiment checks occur and how many events are logged on an hourly basis in the **Diagnostics** tab.
+
 
-As experiment progresses, you will see how many experiment checks have been performed on a daily basis in the **Diagnostics** tab.
+## Step 4: Review experiment results
 
-![image](https://user-images.githubusercontent.com/1315028/139113282-e48d8ede-4f51-4a04-ab34-49ea6978040c.png)
+Within 24 hours of starting your experiment, you'll see the cumulative exposures in the **Results** tab.
 
-You will also start to see the cumulative exposures as well as the allocation share of each variant over time in the **Diagnostics** tab..
+![image](https://user-images.githubusercontent.com/1315028/154574304-79b564b3-0833-49d5-bc65-427b59051343.png)
 
-![image](https://user-images.githubusercontent.com/1315028/139113413-87c9a812-679e-4bb5-af43-52a71a35280b.png)
+You will also start to see the key results for each test group compared to the control in the **Results** tab. 
 
-You will also start to see the key results of how each variant is performing with respect to the control in the **Results** tab. You can also review results to compare one variant against another variant.
+![image](https://user-images.githubusercontent.com/1315028/154574509-adde9b5c-5152-4c91-9e14-bb2c5615b808.png)
 
-![image](https://user-images.githubusercontent.com/1315028/148865822-ba4ad9ef-d1a6-44f6-95c0-24e973d4d280.png)
+In the **Metric Lifts** panel, you can see the full picture of how all your tagged metrics have shifted in the experiment.
 
-In the **Metric Lifts** panel, you will also see whether any of the metric shifts are statistically significant.
+![image](https://user-images.githubusercontent.com/1315028/154574843-480b16b6-2785-4246-87e2-afae03ff358f.png)
 
-![image](https://user-images.githubusercontent.com/1315028/148865844-a3240b51-50b4-4f8f-a0b4-267b5f0ff317.png)
+Once you start to see statistically significant results, either green (positive) or red (negative), you're getting ready to make a decision about whether to ship the test variant. 
 
-At this point, you're ready to make a decision about whether to ship a given variant. Alternatively, you can choose to ramp up the experiment to a larger user base by increasing the allocation in the **Setup** tab. This can help you get more data and build more confidence before you decide.
+Alternatively, you can choose to ramp up the experiment to a larger user base by increasing the allocation in the **Setup** tab. This could help you get more data and build more confidence before you decide to ship. 
```

Comment:
"We recommend rolling out from 2% -> 10% -> 50%" 

From https://docs.statsig.com/feature-gates/best-practices